src/pages/home/report/PureReportActionItem.tsx

```diff
@@ -814,6 +814,8 @@ function PureReportActionItem({
         // Show the MoneyRequestPreview for when expense is present
         if (isIOURequestReportAction(action)) {
             const isSplitInGroupChat = moneyRequestActionType === CONST.IOU.REPORT_ACTION_TYPE.SPLIT && report?.chatType === CONST.REPORT.CHAT_TYPE.GROUP;
+            const isSplitScanWithNoAmount = moneyRequestActionType === CONST.IOU.REPORT_ACTION_TYPE.SPLIT && moneyRequestOriginalMessage?.amount === 0;
```

Comment:
I noticed that if we scan an invalid receipt, then the amount stays 0 and impossible for user to input it manually, so we need to show it when the amount is 0.